.golangci.yml

```diff
@@ -21,23 +32,22 @@ linters-settings:
       - whyNoLint
       - wrapperFunc
       - unnamedResult
-  gocognit:
-    min-complexity: 30
+    settings:
+      hugeParam:
+        sizeThreshold: 512
+      rangeValCopy:
+        sizeThreshold: 512
+  gocyclo:
+    min-complexity: 15
   goimports:
-    local-prefixes: github.com/Mellanox/ib-sriov-cni
+    local-prefixes: github.com/k8snetworkplumbingwg/sriov-network-device-plugin
```

Comment:
should be `github.com/k8snetworkplumbingwg/ib-sriov-cni`

Reply:
fixing, thx !!